src/Sandbox.wsb

```diff
@@ -16,4 +16,6 @@
     <AudioInput>Disable</AudioInput>
     <VideoInput>Disable</VideoInput>
     <ProtectedClient>Enable</ProtectedClient>
+    <ClipboardRedirection>False</ClipboardRedirection>

```

Comment:
I'm surprised that adding that actually prevents detection, but as long as it works, a simple solution is great.